### ISSUE TITLE
Creating an address space should allow you to create the namespace for it

### ISSUE THREAD

**Issue by lulf** [EnMasseProject/enmasse#2730]
If you don't have the project for your address space created already when using the console, you have to create the project first and then go back to the console and select the project.

It would be nice to follow the same patterns as with the service catalog, where you were able to create a project instead of selecting an existing one.

**Comment by vbusch** [comment]
Yes, it would be nice if you could create a project, from the same spot you select the project.
